beacon-chain/blockchain/service.go

```diff
@@ -68,6 +68,24 @@ func (c *ChainService) Stop() error {
 	return nil
 }
 
+// HasStoredState checks if there is any chaindata saved in the local db.
+func (c *ChainService) HasStoredState() (bool, error) {
```

Comment:
So basically just checks if there is anything related to state that has been persisted before